sechub-solution/docker/SecHub-Debian.dockerfile

```
@@ -13,7 +13,7 @@ ARG BASE_IMAGE
 ARG TARGETARCH
 
 # Build args
-ARG BUILD_TYPE="download"
+ARG BUILD_TYPE="copy"
```

Why is this changed? I suggest to keep "download".
```suggestion
ARG BUILD_TYPE="download"
```